bld/namelist_files/namelist_definition.xml

```diff
@@ -3324,6 +3330,32 @@ Convective adjustment timescale in units of (s)
 Default: 3600.0 s
 </entry>
 
+<entry id="zmconv_tht_thermo" type="logical" category="conv"
+       group="zmconv_nl" valid_values="" >
+tht: use moist td to compute plume-ensemble properties
+Default: .false.
+</entry>
+<entry id="zmconv_retrigger"  type="logical" category="conv"
+       group="zmconv_nl" valid_values="" >
+tht: iterate plume-ensemble computation and trigger functions
+Default: .false. 
+</entry>
+<entry id="zmconv_entrmn"     type="real"    category="conv"
+       group="zmconv_nl" valid_values="" >
+tht: previously undeclared par: max entr. rate for plume-ens
+Default: 2e-4
+</entry>
+<entry id="zmconv_alfadet"    type="real"    category="conv"
+       group="zmconv_nl" valid_values="" >
+tht: previously undeclared param: detrainment/entrainment
```

Comment:
Please spell this out without abbreviations and unhelpful comments. Also, what are the units?